### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: src/advert_phone_numbers.py [hunk 1]
```diff
@@ -10,29 +10,31 @@
...
+
+        advert_phone_numbers = set()
+        
         for td in data:
             for plan in PipelineConfiguration.RQA_CODING_PLANS:
-                    if (plan.binary_coded_field is not None and td[plan.binary_coded_field]["CodeID"] != "code-NOP-4eb70633") \
-                        or td[plan.coded_field][0]["CodeID"] != "code-NOP-4eb70633":
-                        advert_phone_numbers.add(phone_number_uuid_table.get_phone(td['uid']))
-        
+                if plan.binary_code_scheme is not None and td[plan.binary_coded_field]["CodeID"] not in ["code-NOP-4eb70633", "code-STOP-08b832a8", "code-NA-f93d3eb7","code-NR-5e3eee23"]:
```
Comment (lukechurch) [inline src/advert_phone_numbers.py]: We should extract this list into a constant

File: src/advert_phone_numbers.py [hunk 2]
```diff
@@ -10,29 +10,31 @@
...
+                    advert_phone_numbers.add(phone_number_uuid_table.get_phone(td['uid']))
+                if td[plan.coded_field][0]["CodeID"] not in ["code-NOP-4eb70633", "code-STOP-08b832a8", "code-NA-f93d3eb7","code-NR-5e3eee23"]:
```
Comment (lukechurch) [inline src/advert_phone_numbers.py]: @as2388 Is this is a reasonable way of checking a code?

File: src/advert_phone_numbers.py (hunk 1 above)
Comment (IsaackMwenda) [reply src/advert_phone_numbers.py]: List moved to pipeline_configuration.py as constant https://github.com/AfricasVoices/Project-ICRAF/pull/16/commits/95c32388bf14c357e30a904ae3ee78dabf60587e.

**Comment by as2388** [comment]
@IsaackMwenda could you:
 - fix the conflicting changes (by merging master into this branch)
 - state exactly what is "WIP"
 - confirm that this PR isn't depending on #14

**Pull request merged**